File: week-3/day-1/prim.py
temp = input('adj meg egy szamot\n')
num = int(temp)

has_divisor = 0
i = 2
while i < num and not has_divisor:
    if num % i == 0:
        has_divisor = 1
    i += 1

if has_divisor or num == 0 or num == 1:
    print('The number is not prime')
else:
    print('The number is a prime')

File: holidaJS/prim.js
function isPrim(num) {
    var hasDivisor = 0;
    var i = 2;
    while (i < num) {
        if (num % i ==0) {
            hasDivisor = 1;
            break
        }
        else {
            i++
        }
    }

    if(hasDivisor ==0 && num != 0 && num != 1) {
        console.log("it's prim!")
    }else {
        console.log("it's not prim!")
    }
}

isPrim(2)
